osu.Game/Overlays/Profile/ProfileHeader.cs

```diff
@@ -357,16 +357,13 @@ private void loadUser()
 
             infoTextLeft.NewParagraph();
 
-            if (user.JoinDate.ToUniversalTime().Year < 2008)
-            {
-                infoTextLeft.AddText("Here since the beginning", boldItalic);
-            }
-            else
+            if (user.JoinDate.ToUniversalTime().Year >= 2008)
```

Comment:
Would it be possible to move "Joined" into `DrawableJoinDate` as well so you don't need to have this here anymore?

Reply:
The thing is, I forgot that "Here since the beginning" should be formatted in _lightText_, so this IF will be here anyway. "Joined" part of the text also doesn't give a tooltip on the web, so that would also mean de-unification.